spec/yaml/swiftnav/sbp/observation.yaml

```diff
@@ -33,22 +33,6 @@ definitions:
             units: week
             desc: GPS week number
 
- - CarrierPhase:
-    short_desc: GPS carrier phase measurement.
-    desc: |
-      Carrier phase measurement in cycles represented as a 40-bit
-      fixed point number with Q32.8 layout, i.e. 32-bits of whole
-      cycles and 8-bits of fractional cycles.
-    fields:
-        - i:
-            type: s32
-            units: cycles
-            desc: Carrier phase whole cycles
-        - f:
-            type: u8
-            units: cycles / 256
-            desc: Carrier phase fractional part
-
  - ObservationHeader:
```

Comment:
@mfine, neither the interface nor content of the fields is changing. What is changing is the way those fields in the observations are computed, so I don't think that needs to change.